soc/arm/ti_k3/j721e/linker.ld

```diff
@@ -0,0 +1,20 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/arm/aarch32/cortex_a_r/scripts/linker.ld>
+
+SECTIONS
+{
+    SECTION_PROLOGUE(.resource_table,,)
+    {
+        KEEP(*(.resource_table*))
+    } > RAM
+
+    .log_shared_mem :
+    {
+        *(.log_shared_mem*)
```

Comment:
Are you using this log buffer? or have plans to use this in future?

Reply:
In this commits this log buffer is not used. In future it can be used for sending logs to linux.

Reply:
Can we drop the  RAM_CONSOLE section, I don't see why this should be part of platform support especially since it is not being used now.